XpringKit/JavaScript/JavaScriptUtils.swift

```diff
@@ -80,6 +80,9 @@ internal class JavaScriptUtils {
     var arguments: [Any] = [ classicAddress ]
     if tag != nil {
       arguments.append(tag as Any)
+    } else {
+      let undefined = JavaScriptLoader.load("undefined", from: JSValue(undefinedIn: <#T##JSContext!#>))
```

Comment:
I think that if you just pass nil it will evaluate to `undefined` when converted to JS. See table 1 here: https://developer.apple.com/documentation/javascriptcore/jsvalue

Reply:
Note: Keefer and I discovered that this apple documentation is wrong, and `nil` was actually converting to `0` in JS.  We instead use `JSValue(undefinedIn: JSContext)` to represent an absent argument.